File: app/src/main/java/com/v2/desafionubank/di/ActivityContext.java
package com.v2.desafionubank.di;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import javax.inject.Qualifier;

/**
 * Created by CaioSChristino on 05/02/18.
 */

@Qualifier
@Retention(RetentionPolicy.RUNTIME)
public @interface ActivityContext {

}

File: app/src/main/java/com/v2/desafionubank/di/ActivityContext.kt
package com.v2.desafionubank.di

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy
import javax.inject.Qualifier

/**
 * Created by csanchez on 19/04/2018.
 */

@Qualifier
@Retention(RetentionPolicy.RUNTIME)
annotation class ActivityContext